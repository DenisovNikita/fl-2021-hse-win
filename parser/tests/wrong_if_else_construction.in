begin function main 00
begin body
    var int10 y = 00
    begin if
    y == 021
    begin body
        skip
    end body
    иначе
    begin body
        skip
    end body
    end if
end body
end function