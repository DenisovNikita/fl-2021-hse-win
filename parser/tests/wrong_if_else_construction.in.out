Error in program.
Expect construction:

begin if
condition
body
else ...

But found:

begin if
condition
body
иначе ...
